### PR TITLE
Fix typos and grammar in FIX4.2 and 4.4 specs

### PR DESCRIPTION
**Pull request by michaelboyles**

### CODE REVIEW

**Comment by kleihan** [comment]
@michaelboyles sorry for the delay, this is the first pull request we received for this and had to discuss our maintenance process for this and if we want to support this path at all. So far, we only offered the [FIX discussion forum](https://forum.fixtrading.org) as a way for users to point to errors and omissions in (legacy) FIX versions, e.g. https://forum.fixtrading.org/t/fix-standard-errors-and-omissions-2022/16385. This would then feed into the standard process, e.g. https://forum.fixtrading.org/t/public-comment-period-fix-protocol-gap-analysis-errors-and-omissions-2022/17066. That can come with quite an overhead when it is only about simple typos.

Many thanks for the issues you identified in the Orchestra representations of the legacy FIX versions 4.2 and 4.4. We want to use it to enhance the quality of the text in the files. We reviewed your proposed changes and disagree with the general change from "whether" to "whether or not". We do not think this is material and in general try to avoid textual changes in the legacy versions as much possible. I will also make some specific comments inline.

File: FIX Standard/OrchestraFIX44.xml [hunk 1]
```diff
@@ -37252,7 +37252,7 @@
 				<fixr:groupRef id="1014" added="FIX.4.4">
 					<fixr:annotation>
 						<fixr:documentation>
-         Insert here here the set of "Position Amount Data" fields defined in "Common Components of Application Messages"
+         The set of "Position Amount Data" fields defined in "Common Components of Application Messages"
```
Comment (kleihan) [inline FIX Standard/OrchestraFIX44.xml:37255]: FIX Latest has over 200 places where we use the text "Insert here..." and we do not want to change that. Agree to remove the duplication "here here..."

File: FIX Standard/OrchestraFIX44.xml [hunk 2]
```diff
@@ -39140,7 +39140,7 @@
 				<fixr:groupRef id="1014" presence="required" added="FIX.4.4">
 					<fixr:annotation>
 						<fixr:documentation>
-         Insert here here the set of "Position Amount Data" fields defined in "Common Components of Application Messages"
+         The set of "Position Amount Data" fields defined in "Common Components of Application Messages"
```
Comment (kleihan) [inline FIX Standard/OrchestraFIX44.xml:39143]: FIX Latest has over 200 places where we use the text "Insert here..." and we do not want to change that. Agree to remove the duplication "here here..."

File: FIX Standard/OrchestraFIX44.xml [hunk 3]
```diff
@@ -39637,7 +39637,7 @@
...
```
Comment (kleihan) [inline FIX Standard/OrchestraFIX44.xml:39640]: FIX Latest has over 200 places where we use the text "Insert here..." and we do not want to change that. Agree to remove the duplication "here here..."

File: FIX Standard/OrchestraFIX44.xml [hunk 4]
```diff
@@ -40949,7 +40949,7 @@
...
```
Comment (kleihan) [inline FIX Standard/OrchestraFIX44.xml:40952]: FIX Latest has over 200 places where we use the text "Insert here..." and we do not want to change that. Agree to remove the duplication "here here..."

File: FIX Standard/OrchestraFIX44.xml [hunk 5]
```diff
@@ -42659,7 +42659,7 @@
 				<fixr:fieldRef id="96" added="FIX.4.4">
 					<fixr:annotation>
 						<fixr:documentation>
-         Can be used to hand structures etc to other API’s etc
+         Can be used to hand structures etc. to other APIs etc
```
Comment (kleihan) [inline FIX Standard/OrchestraFIX44.xml:42662]: Please also add a period after the second occurrence of "etc".

**Comment by michaelboyles** [comment]
Thanks for the review. These were mostly auto-generated, I just eyeballed the changes and tweaked them. So I don't have strong feelings about the things you disagreed with changing.

I'll update as per your comments

**Review by kleihan — APPROVED** [review]
Thank you very much for your contribution!

**Pull request merged**